hathor/indexes/memory_info_index.py

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Hathor Labs
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from typing import Optional
+
+from hathor.indexes.info_index import InfoIndex
+from hathor.transaction import BaseTransaction
+
+
+class MemoryInfoIndex(InfoIndex):
+    def __init__(self):
+        self._block_count = 0
+        self._tx_count = 0
+        self._first_timestamp = 0
+        self._latest_timestamp = 0
+
+    def init_start(self) -> None:
+        self.force_clear()
+
+    def get_db_name(self) -> Optional[str]:
+        return None
+
+    def force_clear(self) -> None:
+        from hathor.transaction.genesis import BLOCK_GENESIS, TX_GENESIS2
+        self._block_count = 1
+        self._tx_count = 2
+        self._first_timestamp = BLOCK_GENESIS.timestamp
```

Comment:
Currently `hathor.transaction.genesis` hardcodes the order, it will always be "block, tx1, tx2", I think it will be enough to point this out with an XXX comment.